domains/marawanxmamdouh.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "marawanxmamdouh",
+    "email": "marawanxmamdouh@gmail.com"
+  },
+  "record": {
+    "CNAME": "linkedin.com/in/marawanxmamdouh"
```

```suggestion
    "CNAME": "linkedin.com/in/marawanxmamdouh"
```
You do not own linkin.com and slashes are not allowed in CNAME records. Come back once you've read the documentation.